File: diordnA_htiw/PracticeProject/android/app/src/main/java/com/practiceproject/SignupPage.java
package com.practiceproject;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class SignupPage extends Activity {
    public static String Name="",Password="",Email="";
    @Override
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.singup_main);

        EditText name=(EditText) findViewById(R.id.name);
        EditText Email=(EditText) findViewById(R.id.email);
        EditText Password=(EditText) findViewById(R.id.password);
        Intent obj= new Intent(getApplicationContext(),LoginPage.class);
        Button but=(Button) findViewById(R.id.button);
        Button but1=(Button) findViewById(R.id.button1);

        but.setOnClickListener(view -> {
            //code
            SignupPage.Name=name.getText().toString();
            String email=Email.getText().toString();
            SignupPage.Email=email;
            System.out.println(email+"Email is"+SignupPage.Email);
            String pass=Password.getText().toString();
            SignupPage.Password=pass;
            startActivity(obj);
        });
        but1.setOnClickListener((view)->{

            startActivity(obj);
        });

    }
}

File: diordnA_htiw/PracticeProject/android/app/src/main/java/com/kotlineSampleProject/SignupPage.kt
//package com.practiceproject
//
//import android.app.Activity
//import android.content.Intent
//import android.os.Bundle
//import android.view.View
//import android.widget.Button
//import android.widget.EditText
//
//class SignupPage : Activity() {
//    var Name="";
//    var Email="";
//    var Password="";
//    public override fun onCreate(savedInstanceState: Bundle?) {
//        super.onCreate(savedInstanceState)
//        setContentView(R.layout.singup_main)
//        val name = findViewById<View>(R.id.name) as EditText
//        val email = findViewById<View>(R.id.email) as EditText
//        val password = findViewById<View>(R.id.password) as EditText
//        val obj = Intent(applicationContext, LoginPage::class.java)
//        val but = findViewById<View>(R.id.button) as Button
//        val but1 = findViewById<View>(R.id.button1) as Button
//        but.setOnClickListener { view: View? ->
//            //code
//            Name = name.text.toString()
//            val email = email.text.toString()
//            Email = email
//            println(email + "Email is" +  Email)
//            val pass = password.text.toString()
//             Password = pass
//            startActivity(obj)
//        }
//        but1.setOnClickListener { view: View? ->
//            startActivity(
//                obj
//            )
//        }
//    }
//
//
//}